wiki/SawStop-Tablesaw.md

```diff
@@ -0,0 +1,33 @@
+- [Safety](#safety)
+- [General Use](#general-use)
+
+## Safety
+The SawStop 10” Professional Cabinet Saw is equipped with several safety features to help protect users from potential accidents. Please adhere to the following safety guidelines while operating the saw:
+
+- Do **not** wear gloves, loose clothing, or jewelry while operating the saw. These items can get caught in the blade and cause serious injury.
+- Keep your hands away from the blade and use push sticks or push blocks to guide the material through the saw. These items are in the black drawers under the saw to the right.
+- When making cuts, stand to the side of the blade and not directly behind it. The blade can kick material back and cause serious injury.
+- Familiarize yourself with the emergency stop paddle switch and its location for immediate shutdown in case of an emergency. It is common practice to stand in a way the switch is able to be hit with your thigh or knee.
+- The main dust collection unit **must** be turned on and connected when the tablesaw is in use.
+- Ensure the riving knife is properly installed and adjusted for the specific cutting operation. Only remove it if necessary.
+- It is worth wearing appropriate safety equipment, including safety glasses, hearing protection, and a dust mask.
+
+## General Use
+The SawStop 10” Professional Cabinet Saw is a versatile tool capable of handling various cutting tasks. Here are some general instructions for using the saw effectively:
+
+1. Prepare the saw for operation by ensuring the angle of the blade (check squareness on straight cuts) and the blade is securely tightened. 
+2. For rip cuts, adjust the fence to the desired distance from the blade. For cross cuts, use the cross-cut sled or the miter gauge to guide the material through the blade. The cross cut sled is located under the outfeed table and the miter gauge is attached to the right side of the table saw.
+3. Measure and mark the material to be cut, ensuring the dimensions are accurate.
+4. Adjust the blade height and tilt angle according to the desired cut. Common practice is to set the blade height to 1/8" above the material being cut.
+5. If using a fence, position the material against the fence, making sure it is firmly held in place. It is paramount to always be pushing towards the fence while pushing the material forward, otherwise a straight cut will not be achieved.
+6. Ensure the dust collection unit is turned on and the tube is connected to the saw.
+6. Start the saw and guide the material through the blade using appropriate techniques and safety accessories.
+7. Once the cut is complete, turn off the saw and wait for the blade to come to a complete stop before removing the cut pieces.
+
+If you have any further questions or need additional information, please feel free to ask in the Discord or if anyone is present in the space.
+
+## User Manual
+* [Manual](https://github.com/redmountainmakers/wiki/blob/main/wiki/pdfs/SawStop-Tablesaw.pdf)
```

Comment:
Use a relative link here.  pdfs/SawStop-Tablesaw.pdf   instead of https://github.com/redmountainmakers/wiki/blob/main/wiki/pdfs/SawStop-Tablesaw.pdf